lib/ansible/plugins/doc_fragments/ovirt_info.py

```diff
@@ -12,7 +12,7 @@ class ModuleDocFragment(object):
     fetch_nested:
         description:
             - If I(yes) the module will fetch additional data from the API.
-            - It will fetch IDs of the VMs disks, snapshots, etc. User can configure to fetch other
+            - It will fetch IDs of first API level. User can configure to fetch other
```

Comment:
It will fetch only IDs of nested entity. It doesn't fetch multiple levels of nested attributes. Only the attributes of the current entity...